.github/workflows/pr-build.yml

```diff
@@ -0,0 +1,48 @@
+name: PR Build Compose
+
+on:
+  pull_request:
+    paths-ignore:
+      - "**.md"
+      - ".idea/**"
+
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          # Make sure the release step uses its own credentials:
+          # https://github.com/cycjimmy/semantic-release-action#private-packages
+          persist-credentials: false
+          fetch-depth: 0
+
+      - name: Set up JDK 17
+        uses: actions/setup-java@v3
+        with:
+          java-version: '17'
+          distribution: 'temurin'
+          cache: gradle
```

Comment:
I don't think this is necessary, the runner already comes with JDK 17.

Reply:
The build is failing and saying that it requires Java 17

Reply:
Odd, the docs specify the runner comes with JDK 17: https://github.com/actions/runner-images/blob/main/images/linux/Ubuntu2204-Readme.md#java

Reply:
As you can see from [this run](https://github.com/ReVanced/revanced-manager/actions/runs/6067130741/job/16458436540) it failed to find JDK 17.